wbooks/src/app/screens/book-detail/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import {  
+  View, 
+  Image,
+  Text,
+  TouchableOpacity
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class BookDetail extends Component {
+	 
+	render() {
+
+    const book = this.props.navigation.getParam('book');
+
+		return (
+      <View style={styles.container}>
+        <View style={styles.cardDetail}>
+          <View style={styles.cardContainer}>
+            <Image style={styles.image}
+            source={{uri: book.image_url}} />
+            <View style={styles.textContainer}>
+              <Text style={styles.bookTitle}>{book.title}</Text>    
+              <Text style={styles.bookAvailability}>Available</Text>    
+              <Text style={styles.bookAuthor}>{book.author}</Text>    
+              <Text style={styles.bookYear}>{book.year}</Text>    
+              <Text style={styles.bookGenre}>{book.genre}</Text>    
+            </View>
+          </View>
+          <View>
+            <TouchableOpacity style={styles.whishListButton}>
```

Comment:
Every Touchable must have an onPress function

Reply:
if you want an empty function you can add this: `() => {}`

Reply:
ok, done!